tests/functional/g/generic_alias/generic_alias_side_effects.rc

```diff
@@ -1,2 +0,0 @@
-[testoptions]
```

Comment:
```suggestion
py-version=3.8
```

Reply:
I see the benefit of doing that when removing the `max_pyver=3.8` setting, and that was done in #8609. But what would be the benefit of doing it when removing `min_pyver`? When testing on 3.8, there is no difference, and when testing on 3.9, we would be saying "test as if on 3.8", but again, no difference?

Reply:
I found this to be a real brain-teaser :-}

Reply:
I thought again about it, unless we're testing something specific to python 3.8 (like the need to have from __future__ import annotations``) and if we're not modifying the test itself then the py-version is useless in the conf.